apps/ui/src/api/useGetApplicationList.ts

```diff
@@ -22,14 +22,41 @@ import { useTranslation } from 'react-i18next';
 
 import { fetch } from '@/global/FetchClient';
 import { ApplicationList, ServerError } from '@/global/types';
+import { ApplicationStateValues } from '@pcgl-daco/data-model/src/types';
 
-const useGetApplicationList = (id?: string | number) => {
+export interface ApplicationListSortingOptions {
+	direction: 'desc' | 'asc';
+	column: 'user_id' | 'id' | 'created_at' | 'updated_at' | 'state' | 'approved_at' | 'expires_at';
+}
+interface ApplicationListParams {
+	userId: string;
+	state?: ApplicationStateValues[];
+	sort?: ApplicationListSortingOptions[];
+	page?: number;
+	pageSize?: number;
+}
```

Comment:
So I think ApplicationListSortingParams could be moved to the shared `types` package

ApplicationListSortingOptions essentially duplicates this: https://github.com/Pan-Canadian-Genome-Library/daco/blob/main/apps/api/src/service/types.ts#L40

Main difference is the api definition allows sorting by contents but that could easily be omitted or a separate type created for 'SortingColumns'

Reply:
Discussed offline with @surajgoraya , we need to resolve disconnect between Drizzle Schema / Shared Types definitions to solve this properly and prevent further Type duplication. With Zod recently added we should be able to unify these.

I adding this as an action item to: https://github.com/orgs/Pan-Canadian-Genome-Library/projects/1/views/1?pane=issue&itemId=95531183&issue=Pan-Canadian-Genome-Library%7Cdaco%7C178
(and also including a link to this comment, so @surajgoraya leave this discussion unresolved)